test/modules/charging-import/controller.js

```diff
@@ -1,12 +1,12 @@
 const { test, experiment, beforeEach, afterEach } = exports.lab = require('@hapi/lab').script();
 const { expect } = require('@hapi/code');
 const controller = require('../../../src/modules/charging-import/controller');
-const chargingImport = require('../../../src/modules/charging-import/lib/import');
+const chargeVersionsJob = require('../../../src/modules/charging-import/jobs/charge-versions');
 const sandbox = require('sinon').createSandbox();
 
 experiment('modules/charging-import/controller.js', () => {
   beforeEach(async () => {
-    sandbox.stub(chargingImport, 'importChargingData');
+    // sandbox.stub(chargingImport, 'importChargingData');
```

Comment:
If this is intentionally commented out, should we take it out?

Reply:
Deleted - thanks!